proof/invariant-abstract/AARCH64/ArchKHeap_AI.thy

```diff
@@ -828,6 +828,34 @@ lemma valid_arch_state_lift_aobj_at:
 end
 end
 
+\<comment> \<open>FIXME arch-split: if tcb_at and related terms were moved to InvariantsPre_AI then this lemma could go in
+     ArchInvariants_AI.\<close>
+lemma current_fpu_owner_Some_tcb_at:
+  "\<lbrakk>arm_current_fpu_owner (arch_state s) = Some p; valid_cur_fpu s\<rbrakk> \<Longrightarrow> tcb_at p s"
+  by (auto simp: valid_cur_fpu_defs is_tcb)
+
+\<comment> \<open>FIXME arch-split: is_tcb_cur_fpu can't be defined like this because arch_tcb_at isn't defined until
+     Invariants_AI.\<close>
```

Comment:
any reason for this novelty in indentation (and above)? it's showing up as aligning with "X" in "FIXME"

Reply:
No reason, I've tweaked it now. My best guess is that it was a combination of not paying much attention due to it being in a FIXME comment and vaguely wanting to avoid the long hanging indentation.